File: 장승민/Java/210112_classPractice3/src/com/kb/practice3/testarray/TestArray.java
package com.kb.practice3.testarray;

public class TestArray {

	private int sum;
	private double avg;

	public int sum(int[] arr) {

		for (int i = 0; i < arr.length; i++) {
			sum += arr[i];
		}
		return sum;
	}

	public double avg(int[] arr) {

		avg = (double)sum/arr.length;
		return avg;
	}

	public TestArray(int[]arr) {
		System.out.println(sum(arr));
		System.out.println(avg(arr));
	}

}

File: 강병수/C#/210208/Problem3/Problem3/TestArray.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Problem3
{
    class TestArray
    {

        int sum;
        double avr;
        int[] arr;

        public TestArray(int[] arr)
        {
            this.arr = arr;
            Sum();
            Avr();
        }

        public void Sum()
        {
            for (int i = 0; i < arr.Length; i++)
                sum += arr[i];
            Console.WriteLine("합계: " + sum);
        }

        public void Avr()
        {
            avr = sum / 10;
            Console.WriteLine("평균: " + avr);
        }
    }
}
